### ISSUE TITLE
Feature: Implement `kubernetes.io/arch` labels for node templates in Cluster Autoscaler

### ISSUE THREAD

**Issue by MiroslavRepka** [berops/claudie#665]
Can be worked on after #644 is merged.

### Motivation

Cluster Autoscaler requests a template of a potential new node, which includes all labels it will have. Currently, this template does not include the `kubernetes.io/arch` label, as not all cloud libraries included information about architecture type. This issue aims to add this label to the node template.

### Description

The node template in the `node_manager` package should include the `kubernetes.io/arch` label, with the correct value.

### Exit criteria

- [ ] `kubernetes.io/arch` label is now part of the node template

**Comment by MiroslavRepka** [comment]
I think you got a few things mixed up. The template Cluster Autoscaler requests from our Autoscaler adapter is a yaml manifest of node ( `kind: Node`). The terraform template does not play any role here.

**Comment by MiroslavRepka** [comment]
This issue is postponed until Claudie officially supports `arm64` node pools.
